packaging/hip-targets.cmake

```diff
@@ -39,22 +39,25 @@ unset(_expectedTargets)
 
 
 # The installation prefix configured by this project.
-set(_IMPORT_PREFIX "/opt/rocm/hip")
-
+if( DEFINED ENV{ROCM_PATH} )
```

Comment:
An environment variable should not be used here either.

Reply:
Same comments as packaging/hip-targets-release.cmake